requirements.txt

```diff
@@ -18,6 +18,7 @@ lxml==4.6.2
 MarkupSafe==1.1.0
 num2words==0.5.6
 ofxparse==0.19
+openpyxl==3.0.9
```

Comment:
We have to stick to  the [Debian](https://packages.debian.org/bullseye/python3-openpyxl)/[Ubuntu](https://packages.ubuntu.com/focal/python3-openpyxl) version 3.0.3.